test/unit/annotations/noSelf.spec.ts

```diff
@@ -0,0 +1,53 @@
+import * as util from "../../util";
```

Comment:
I think it would make sense to test it along with other context-affecting things, in `test/unit/functions`